tests/unit/Restoration.spec.ts

```diff
@@ -0,0 +1,373 @@
+      streetAddressAdditional: null
+    }
+  }
+}
+const nameTranslations = [
+  {
+    id: '100',
+    name: 'ALIAS ONE',
+    type: 'TRANSLATION'
+  }
+]
+const directors = {
+  directors: [
+    {
+      appointmentDate: '2023-01-01',
+      cessationDate: null,
+      deliveryAddress: {
+        addressCity: 'Victoria',
+        addressCountry: 'CA',
+        addressRegion: 'BC',
+        deliveryInstructions: null,
+        id: 55555,
+        postalCode: 'V5V 5V5',
+        streetAddress: '555 Main Street',
+        streetAddressAdditional: null
+      },
+      mailingAddress: {
+        addressCity: 'Victoria',
+        addressCountry: 'CA',
+        addressRegion: 'BC',
+        deliveryInstructions: null,
+        id: 66666,
+        postalCode: 'V6V 6V6',
+        streetAddress: '666 Main Street',
+        streetAddressAdditional: null
+      },
+      officer: {},
+      role: 'applicant'
+    }
+  ]
+}
+const stateFiling = {
+  header: {
+    name: 'restoration',
+    status: 'COMPLETED'
+  },
+  business: {
+    identifier: 'BC1234567',
+    legalName: '1234567 B.C. LTD.',
+    legalType: 'BEN'
+  },
+  restoration: {
+    expiry: '2023-01-31',
+    type: 'limitedRestoration'
+  }
+}
+const filingFees = {
+  filingFees: 20.0
+}
+
+// init session storage variables
+sessionStorage.setItem('PAY_API_URL', 'https://pay.api.url/')
+sessionStorage.setItem('KEYCLOAK_TOKEN', 'keycloak-token') // anything non-falsy
+
+describe('Restoration component - edit page', () => {
+  const { assign } = window.location
+  let wrapper: any
+
+  beforeAll(async () => {
+    // init store
+    store.state.stateModel.summaryMode = false
+    store.state.stateModel.validationFlags.appValidate = false
+    store.state.stateModel.tombstone.businessId = 'BC1234567' // normally set in App.vue
+    store.state.stateModel.tombstone.keycloakRoles = ['staff'] // normally set in App.vue
+
+    // mock the window.location.assign function
+    delete window.location
+    window.location = { assign: jest.fn() } as any
+
+    // mock services function
+    jest.spyOn((LegalServices as any), 'fetchFilingById').mockImplementation(
+      () => Promise.resolve(filing))
+    jest.spyOn((LegalServices as any), 'fetchBusinessInfo').mockImplementation(
+      () => Promise.resolve(businessInfo))
+    jest.spyOn((AuthServices as any), 'fetchAuthInfo').mockImplementation(
+      () => Promise.resolve(authInfo))
+    jest.spyOn((LegalServices as any), 'fetchAddresses').mockImplementation(
+      () => Promise.resolve(addresses))
+    jest.spyOn((LegalServices as any), 'fetchNameTranslations').mockImplementation(
+      () => Promise.resolve(nameTranslations))
+    jest.spyOn((LegalServices as any), 'fetchDirectors').mockImplementation(
+      () => Promise.resolve(directors))
+    jest.spyOn((LegalServices as any), 'fetchFiling').mockImplementation(
+      () => Promise.resolve(stateFiling))
+    jest.spyOn((PayServices as any), 'fetchFilingFees').mockImplementation(
```

Comment:
^^ Instead of using sinon and mocking the network calls, we can just mock the service methods.

(Tests for the services should mock the network calls.)